app/src/main/java/org/simple/clinic/summary/teleconsultation/contactdoctor/ContactDoctorSheet.kt

```diff
@@ -0,0 +1,75 @@
+package org.simple.clinic.summary.teleconsultation.contactdoctor
+
+import android.app.Activity
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import android.os.Parcelable
+import io.reactivex.disposables.CompositeDisposable
+import kotlinx.android.synthetic.main.sheet_contact_doctor.*
+import org.simple.clinic.R
+import org.simple.clinic.summary.teleconsultation.api.TeleconsultPhoneNumber
+import org.simple.clinic.util.unsafeLazy
+import org.simple.clinic.widgets.BottomSheetActivity
+import org.simple.clinic.widgets.ItemAdapter
+
+class ContactDoctorSheet : BottomSheetActivity() {
+
+  companion object {
+    private const val EXTRA_FACILITY_NAME = "facility_name"
+    private const val EXTRA_PHONE_NUMBERS = "phone_numbers"
+    private const val EXTRA_PHONE_NUMBER = "phone_number"
+
+    fun intent(
+        context: Context,
+        facilityName: String,
```

Comment:
I think we can pass the `Facility` here as part of the arguments, but continue adding just the name to the intent args. This lets us potentially use the facility UUID in future without needing to change all the call sites. What do you think?

Reply:
Yeah, that's a good idea. Will change it to `Facility`